packages/syft/tests/syft/core/tensor/lazy_repeat_array_test.py

```diff
@@ -94,3 +94,10 @@ def test_sum() -> None:
     lazyarray = lazyrepeatarray(data=np.array([1]), shape=array.shape)
 
     assert lazyarray.sum(axis=None).data == array.sum(axis=None)
+
+
+def test_mean() -> None:
+    array = np.array([1, 2, 3])
+    lazyarray = lazyrepeatarray(data=np.array([1, 2, 3]), shape=array.shape)
```

Comment:
lazyrepeatarray.data is usually just passed as a single value (like an integer)

Reply:
Corrected!